src/core/thread/mle.cpp

```diff
@@ -2444,7 +2444,7 @@ otError Mle::SendOrphanAnnounce(void)
 }
 
 #if OPENTHREAD_CONFIG_MLE_LINK_METRICS_ENABLE
```

Comment:
Add missing comment below:
`#endif // OPENTHREAD_CONFIG_MLE_LINK_METRICS_ENABLE`

And similar other cases for this macro.